sport/app/football/conf/context.scala

```diff
@@ -46,19 +59,24 @@ class FootballLifecycle(
     Ideally, we should combine the logic of these two jobs so we only call once. This is tricky to achieve with the current code.
     We intend to move to a stream model so we can achieve this.
      */
-    jobs.schedule("MatchDayAgentRefreshJob", "0 0/5 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every 5 minutes"
+    // 40 minute was chosen for preview to run these jobs after the 35 CompetitionAgentRefreshJobs are finished
+    jobs.schedule("MatchDayAgentRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/5 * * * ?") {
       competitionsService.refreshMatchDay(defaultClock)
     }
 
-    jobs.schedule("MaybeMatchDayAgentRefreshJob", "0 0/1 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every minute"
+    jobs.schedule("MaybeMatchDayAgentRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/1 * * * ?") {
       competitionsService.maybeRefreshLiveMatches(defaultClock)
     }
 
-    jobs.schedule("CompetitionRefreshJob", "0 0/10 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every 10 minutes"
+    jobs.schedule("CompetitionRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/10 * * * ?") {
       competitionsService.refreshCompetitionData()
     }
 
-    jobs.schedule("TeamMapRefreshJob", "0 0/10 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every minute"
+    jobs.schedule("TeamMapRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/10 * * * ?") {
```

Comment:
@arelra we discussed to use `scheduleEvery` method which gives us ability to use interval, but unfortunately as you see, we have 2 different cron rules here, one for preview and one for non-preview. We can't use `scheduleEvery` for the preview version with its current cron definition `0 40 * * * ?`

Our options are: 
- use `scheduleEvery` for non-preview & use `schedule` for preview. the drawback is that we need to check if its preview before we schedule the job, and we'll end up with 2 different calls for scheduling the job
- Change the cron rule for the preview version to be once every hour (at minute 0) to be able to use `scheduleEvery` instead of `0 40 * * * ?` which runs every hour at 40 minutes past the hour

What do you think?

Reply:
Is there another option to keep as-is, so continue to use schedule for both?

Reply:
Yes that's what I kept it with, right now